openedx/features/edly/tests/test_cookies.py

```diff
@@ -0,0 +1,80 @@
+import jwt
+from mock import MagicMock, patch
+
+from django.conf import settings
+from django.http import HttpResponse
+from django.test import RequestFactory, TestCase
+from openedx.features.edly.tests.factories import SiteFactory, EdlySubOrganizationFactory
```

Comment:
Update imports order just like we are doing in `edx-platform`: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#imports-order

```
from django.conf import settings
from django.http import HttpResponse
from django.test import RequestFactory, TestCase

from openedx.features.edly.tests.factories import SiteFactory, EdlySubOrganizationFactory
from openedx.features.edly import cookies as cookies_api
from student.tests.factories import UserFactory
```

Reply:
The import order is already like this. I followed the order from existing `test_cookies.py` file. I only removed the newline.